react-app-drawer-component/my-app/src/App.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './App.css';
+import AppDrawer, { MenuItem } from './AppDrawer';
+import { useState } from 'react';
+import DrawerButton from './DrawerButton';
+import './App.css';
+
+const items: MenuItem[] = [
+  {
+    title: 'Home',
+  },
+  {
+    title: 'About',
+  },
+  {
+    title: 'Get Started',
+  },
+  {
+    title: 'Sign In',
+  },
+];
+
+function App() {
+  const [activeItem, setActiveItem] = useState('Home');
+  const [drawerState, setDrawerState] = useState(false);
```

Comment:
You name this `drawerState` but it's a boolean. I'm not sure what `true` and `false` mean. Reading your code, it looks like `true` means "open". So wouldn't `drawerOpen` or `isOpen` be better names?